logging/README.md

```diff
@@ -0,0 +1,25 @@
+## Logging
+
+This is logging simplification support for in `kotlin-mpp` wrapping the [Hydra-log](https://github.com/PocketByte/kotlin-hydra-log).
+
+### How to
+
+#### WARNING!!!
+* Because of Hydra-log limitation logger can be initialized only once. All subsequent calls to initialize will return first logger used for initialization.
+* If `initLogger(logger: Logger)` was not called before any logging calls, then default logger will be instantiated.
+
+#### Initialization
+
+*Optionally* call `initLogger(logger: Logger): Logger?` with instance of `Logger`. 
+
+If you need to get logger that was used for initialisation, then you can call `logger(): Logger?`
+
+#### Usage
+
+Here's a list of funnctions available for usage:
```

Comment:
If we want to make such a list we can better release a KDoc somewhere.

I'd provide an example of usage, and explain the difference between `debug("hello")` and `debug { "hello"}` the way Hydralog does.